validator/accounts/v2/prompt.go

```diff
@@ -270,11 +281,23 @@ func inputRemoteKeymanagerConfig(cliCtx *cli.Context) (*remote.Config, error) {
 			return nil, err
 		}
 	}
+	crtPath, err := filepath.Abs(expandPath(strings.TrimRight(crt, "\r\n")))
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not determine absolute path for %s", crt)
+	}
+	keyPath, err := filepath.Abs(expandPath(strings.TrimRight(key, "\r\n")))
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not determine absolute path for %s", crt)
+	}
+	caPath, err := filepath.Abs(expandPath(strings.TrimRight(ca, "\r\n")))
```

Comment:
After promptutil is merged, can you remove the TrimRight? Promptutil always handles it.